work/Maura/exercise_1.1.py

```diff
@@ -0,0 +1,14 @@
+import os
+import time
+
+pid = os.fork()
+if pid < 0:
+    print("Error forking")
+    exit(1)
+if pid > 0:
+    print("Parent process, PID:", os.getpid(), ", Child PID:", pid)
+elif pid == 0:
+    print("Child process, PID:", os.getpid())
+    # Execute the command
```

Comment:
its good but is best if here the children should use exec to run sleep and print hello
tip:     command = ["python", "-c", "import time; time.sleep(2); print('Hello')"]